lib/vm/src/instance.rs

```diff
@@ -186,7 +186,7 @@ mod test {
 
         let init_used = orig_gas - instance.get_gas();
         println!("init used: {}", init_used);
-        assert_eq!(init_used, 66_763);
+        assert_eq!(init_used, 66_762);
```

Comment:
Funny how name changes jiggle gas up or down a few points.
This is kind of a hassle, but I do want to make sure the same code run many times on different machines remains the same gas cost.

Reply:
It is not only the name change. This is a fully recompiled contract, using potentially different version of tools in comswasm-opt and a bunch of changes in the cosmwasm crate that are inlined, like e.g. new error handling code branches.

For me two things would have removed most of the pain:
1. Being able to compile a local version of hackatom with local dependencies using cosmwasm-opt
2. Only test one gas per test, such that all the changes are visible in a single CI fail

Reply:
For 1. I use the comments in Cargo.toml.

Comment out the cosmwasm dependencies and uncomment one pointing to your branch.

For the gas cost, yes, I agree.

Reply:
Thanks, I saw and unsed those lines. But still it required committing and pushing broken state where tests don't pass. Latest version of cosmwasm-opt does the trick.